src/telas/Post/index.js

```diff
@@ -30,69 +21,19 @@ export default function Post({ navigation, route }) {
     }
 
     async function atualizar() {
-        const data = { titulo, fonte, descricao, imagem: imagem ? imagem : null };
+        const data = { titulo, fonte, descricao };
```

Comment:
imagino que o data seria dado. Se for o caso, precisa tomar cuidado, pois está misturando pt-br com inglês. O ideal é sempre manter um padrão